content/usage/advanced/index.md

```diff
@@ -268,6 +289,8 @@ The Autopilot Parameters page allows checking and changing the autopilot's confi
 
 - Includes fuzzy searching of names and descriptions, to help find relevant parameters
 - Allows loading parameters from a file, and saving the current parameters to a file
+- Parameter descriptions can be overridden by including a `metatdata_override.json` file
```

Comment:
it would be nice to link to how these are created (the python script that generates the xml/json/rst files)